### PR TITLE
Added About  and Work section using flexbox and grid.

### PR DESCRIPTION
**Pull request by chudisoft**
-[]Added new assets (images / icons) for the home page.
-[]Created HTML Design for the works section of home page.
-[]Created HTML Design for the about section of home page.
-[]Created CSS for for desktop view.

### CODE REVIEW

**Review by Kamba56 — COMMENTED** [review]
Hi @chudisoft

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that **it's readme file contains comments.** Kindly remove this comments that are there to aid you fill the readme file.
image
image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -75,6 +75,330 @@ <h4>
...
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="about">
+      <div class="about-background flex-container">
+        <h2>
+          About Myself
+        </h2>
```
Comment (abdulhamiid) [inline index.html:294]: Figma
image

Your design
image

To make our design as similar to the guideline as possible using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). kindly add the border-radius as indicated above. 💪

File: index.html [hunk 2]
```diff
@@ -75,6 +75,330 @@ <h4>
...
+              <button type="button" onclick="return;">
+                JavaScript
+              </button>
+            </li>
+          </ul>
+          <div>
+            <button class="project-button" type="button"
+              onclick="return;">
+              See project
+            </button>
```
Comment (abdulhamiid) [inline index.html:285]: I would suggest that we add more `padding-top` & bottom for the buttons similar to the Figma design

Your design
image

Figma
image

File: assets/css/style.css [hunk 3]
```diff
@@ -141,12 +130,236 @@ main {
...
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background-color: #fefefe;
+  color: #396df2;
+  border: #396df2 solid 0.1em;
+  margin-bottom: 1em;
+}
```
Comment (abdulhamiid) [inline assets/css/style.css:260]: I would suggest that we add more `padding-top`, `padding bottom` to the buttons similar to the Figma design

Your design
image

Figma
image

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @chudisoft 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Understand how to parse a Figma design to create a UI.
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow

## Required Changes ♻️
_Please kindly check the comments under the review._

Your commit messages 👇
image

To document our work in a professional way, our commit message should be written in the `imperative` tense.

### **Imperative tenses✔️** 
```
Add about work section using flexbox and grid.

Fix Style Linters errors
```
### **IMPORTANT: 🏮** 

As it is not easy to change commit history, kindly correct this behavior in the future. 🛠🤗


## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css [hunk 4]
```diff
@@ -141,12 +130,238 @@ main {
...
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  background-color: #fefefe;
+  color: #396df2;
+  border: #396df2 solid 0.1em;
+  margin-bottom: 1em;
+  padding: 0.5em;
+}
```
Comment (shahadat3669) [inline assets/css/style.css:262]: As the previous reviewer requested, great work on adding the button styles! but please go through his every comment.  your button does not look the same as designed.

|  styles | Your | Design|
| --- | --- |  --- |
|  Hover| image | image  | 
| Active  | image | image  |

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi  @chudisoft , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✅ Descriptive PR title and summary.
✅ Descriptive commit messages.
✅ No linters errors.

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by taldr27 — APPROVED** [review]
# Status: Approved! 🟢 
Hi @chudisoft, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉
✅ Professional and customized README.  🤓
✅ Linters tests passing. 👍
✅ Correct use of GitHub Flow. 🙌
✅ Great job with your cards and about sections. 🎯
✅ Incorporates feedback from previous code reviews. 💯

-----
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Pull request merged**